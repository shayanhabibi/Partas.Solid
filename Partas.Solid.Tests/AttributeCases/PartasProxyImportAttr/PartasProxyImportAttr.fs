module Partas.Solid.Tests.AttributeCases.PartasImportAttr

open Partas.Solid
open Partas.Solid.Router
open Fable.Core

[<Erase; PartasProxyImport("div", "motion","motion-one")>]
type Test() =
    inherit RegularNode()

[<SolidComponent>]
let Bock () =
    Test()

[<SolidComponent>]
let chock () =
    Route()