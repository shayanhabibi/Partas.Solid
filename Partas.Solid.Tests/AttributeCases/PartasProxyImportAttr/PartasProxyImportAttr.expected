import { motion } from "motion-one";
import { Route } from "@solidjs/router";


export function Bock() {
    return <motion.div />;
}

export function chock() {
    return <Route />;
}

